test/peach_test.exs

```diff
@@ -1,40 +1,213 @@
 defmodule PeachTest do
+  # import CSVLixir
   use ExUnit.Case
   doctest Peach
 
-  test "greets the world" do
-    assert Peach.hello() == :world
-  end
-
+  @tag timeout: 600_000
   test "normalise_text" do
-    assert Peach.normalise_text() == :to_do
+    test_data = [
+      ["foo", "foo"],
+      ["\n", "\n"],
+      ["🚼", "🚼"]
```

Comment:
@bduvenhage do you think you could provide some test examples here? Feel free to pop them in this comment and I can add them.